tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1030,6 +1030,11 @@ platform_tests/test_cont_warm_reboot.py:
     conditions:
       - "'dualtor' in topo_name"
 
+platform_tests/test_reboot.py::test_warm_reboot:
```

Comment:
I think same change is also needed for `platform_tests/test_reboot.py::test_fast_reboot`

Reply:
Addressed. 

> I think same change is also needed for `platform_tests/test_reboot.py::test_fast_reboot`